samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_ca.properties

```diff
@@ -437,6 +437,8 @@ seb_exam_keys_upload_info=Per a configuracions pujades amb fitxer les claus de l
 Descarrega la configuraci\u00f3 (alterada) de nou i insereix les claus de l\u2019examen presents en el fitxer.
 seb_exam_keys_upload_published_info=Per favor, insereix les claus de l\u2019examen presents en el fitxer de configuraci\u00f3 que pots descarregar dalt. \
 La configuraci\u00f3 s\u2019ha alterat quan l\u2019examen es va publicar.
+seb_exam_keys_recommended=It is recommended to add exam keys for this type of configuration, because the Safe Exam Browser Configuration can \
+not be validated automatically
```

Comment:
@bgarciaentornos Could you help me with this translation please?

Reply:
```suggestion
seb_exam_keys_recommended=Es recomana afegir claus de prova per a aquest tipus de configuraci\u00f3, ja que la configuraci\u00f3 del Safe Exam Browser \
no pot ser validada autom\u00e0ticament
```

Reply:
Thanks!